### PR TITLE
Fixes #10298 - Fix registry path information for policies

### LINKED ISSUE #10298: Incorrect information about enabling Script Block Logging for PowerShell 7+
### Prerequisites

- [X] **Existing Issue:** Search the existing issues for this repository. If there is an issue that fits your needs do not file a new one. Subscribe, react, or comment on that issue instead.
- [X] **Descriptive Title:** Write the title for this issue as a short synopsis. If possible, provide context. For example, "Typo in `Get-Foo` cmdlet" instead of "Typo."
- [X] **Verify Version:** If there is a mismatch between documentation and the behavior on your system, ensure that the version you are using is the same as the documentation. Check this box if they match or the issue you are reporting is not version specific.

### Links

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_logging_windows?view=powershell-7.3

### Summary

To enable Script Block Logging on PowerShell Core version (7.3, but probably applies for other 7.x versions too), you need to create registry DWORD value 'EnableScriptBlockLogging = 1' in 'HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Policies\Microsoft\PowerShellCore\ScriptBlockLogging' . 

What the documentation says is for Windows PowerShell only.

### Details

Registry and also GPO settings show information for Windows PowerShell instead of PowerShell Core. For Core version, there is an extra ADMX template and the setting location is 👇
image

the registry is 👇
'HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Policies\Microsoft\PowerShellCore\ScriptBlockLogging'

### Suggested Fix

Modify the registry and gpo paths.

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fix registry path information for policies

- Fixes [AB#141625](https://dev.azure.com/msft-skilling/cebd7ef5-4282-448b-9701-88c8637581b7/_workitems/edit/141625)
- Fixes #10298

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: reference/7.2/Microsoft.PowerShell.Core/About/about_Logging_Windows.md [hunk 1]
```diff
@@ -84,22 +77,20 @@ Script Block Logging can be enabled via Group Policy or a registry setting.
 
 ### Using Group Policy
 
-To enable automatic transcription, enable the `Turn on PowerShell Script Block
-Logging` feature in Group Policy through `Administrative Templates -> Windows
-Components -> Windows PowerShell`.
+To enable automatic transcription, enable the **Turn on PowerShell Script Block
+Logging** feature in Group Policy through **Administrative Templates** ->
+**Windows Components** -> **Windows PowerShell**.
```
Comment (michaeltlombardi) [inline reference/7.2/Microsoft.PowerShell.Core/About/about_Logging_Windows.md:82]: If I read the issue correctly, this setting has moved, right?

```suggestion
To enable automatic transcription, enable the **Turn on PowerShell Script Block
Logging** feature in Group Policy through **Administrative Templates** ->
**PowerShell Core**.
```

File: reference/7.3/Microsoft.PowerShell.Core/About/about_Logging_Windows.md [hunk 2]
```diff
@@ -84,22 +77,20 @@ Script Block Logging can be enabled via Group Policy or a registry setting.
 
 ### Using Group Policy
 
-To enable automatic transcription, enable the `Turn on PowerShell Script Block
-Logging` feature in Group Policy through `Administrative Templates -> Windows
-Components -> Windows PowerShell`.
+To enable automatic transcription, enable the **Turn on PowerShell Script Block
+Logging** feature in Group Policy through **Administrative Templates** ->
+**Windows Components** -> **Windows PowerShell**.
```
Comment (michaeltlombardi) [inline reference/7.3/Microsoft.PowerShell.Core/About/about_Logging_Windows.md:82]:
```suggestion
To enable automatic transcription, enable the **Turn on PowerShell Script Block
Logging** feature in Group Policy through **Administrative Templates** ->
**PowerShell Core**.
```

File: reference/7.4/Microsoft.PowerShell.Core/About/about_Logging_Windows.md [hunk 3]
```diff
@@ -84,22 +77,20 @@ Script Block Logging can be enabled via Group Policy or a registry setting.
 
 ### Using Group Policy
 
-To enable automatic transcription, enable the `Turn on PowerShell Script Block
-Logging` feature in Group Policy through `Administrative Templates -> Windows
-Components -> Windows PowerShell`.
+To enable automatic transcription, enable the **Turn on PowerShell Script Block
+Logging** feature in Group Policy through **Administrative Templates** ->
+**Windows Components** -> **Windows PowerShell**.
```
Comment (michaeltlombardi) [inline reference/7.4/Microsoft.PowerShell.Core/About/about_Logging_Windows.md:82]:
```suggestion
To enable automatic transcription, enable the **Turn on PowerShell Script Block
Logging** feature in Group Policy through **Administrative Templates** ->
**PowerShell Core**.
```

**Review by michaeltlombardi — COMMENTED** [review]
One nit, one suggestion where I might've misunderstood the change.

File: reference/7.3/Microsoft.PowerShell.Core/About/about_Logging_Windows.md [hunk 4]
```diff
@@ -84,22 +77,20 @@ Script Block Logging can be enabled via Group Policy or a registry setting.
...
 Run the following function:
 
 ```powershell
-function Enable-PSScriptBlockLogging
-{
-    $basePath = 'HKLM:\Software\Policies\Microsoft\Windows' +
-      '\PowerShell\ScriptBlockLogging'
+function Enable-PSScriptBlockLogging {
+    $basePath = 'HKLM:\Software\Policies\Microsoft\' +
+        'PowerShellCore\ScriptBlockLogging'
```
Comment (sdwheeler) [inline reference/7.3/Microsoft.PowerShell.Core/About/about_Logging_Windows.md:91]:
```suggestion
    $basePath = @(
        'HKLM:\Software\Policies\Microsoft'
        'PowerShellCore\ScriptBlockLogging'
    ) -join '\'
```

File: reference/7.4/Microsoft.PowerShell.Core/About/about_Logging_Windows.md [hunk 5]
```diff
@@ -84,22 +77,20 @@ Script Block Logging can be enabled via Group Policy or a registry setting.
...
 Run the following function:
 
 ```powershell
-function Enable-PSScriptBlockLogging
-{
-    $basePath = 'HKLM:\Software\Policies\Microsoft\Windows' +
-      '\PowerShell\ScriptBlockLogging'
+function Enable-PSScriptBlockLogging {
+    $basePath = 'HKLM:\Software\Policies\Microsoft\' +
+        'PowerShellCore\ScriptBlockLogging'
```
Comment (sdwheeler) [inline reference/7.4/Microsoft.PowerShell.Core/About/about_Logging_Windows.md:91]:
```suggestion
    $basePath = @(
        'HKLM:\Software\Policies\Microsoft'
        'PowerShellCore\ScriptBlockLogging'
    ) -join '\'
```

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Logging.md [hunk 6]
```diff
@@ -58,22 +58,20 @@ Script Block Logging can be enabled via Group Policy or a registry setting.
...
 ### Using the Registry
 
 Run the following function:
 
 ```powershell
-function Enable-PSScriptBlockLogging
-{
+function Enable-PSScriptBlockLogging {
     $basePath = 'HKLM:\Software\Policies\Microsoft\Windows' +
       '\PowerShell\ScriptBlockLogging'
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Logging.md:72]:
```suggestion
    $basePath = @(
        'HKLM:\Software\Policies\Microsoft\Windows'
        'PowerShell\ScriptBlockLogging'
    ) -join '\'
```

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**